package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "remove-stale-projects",
+  "version": "0.0.1",
+  "description": "Determine stale projects and remove them",
+  "main": "index.js",
+  "scripts": {
+    "build": "ncc build index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "dependencies": {
+    "@actions/core": "^1.6.0",
+    "@actions/github": "^5.0.0"
```

I think it is not necessary to pin them here as we do use the lockfile. 
```suggestion
    "@actions/core": "*",
    "@actions/github": "*"
```